aten/src/ATen/native/native_functions.yaml

```diff
@@ -3230,6 +3230,31 @@
   variants: function, method
   device_guard: False
 
+- func: hsplit.int(Tensor(a) self, int indices_or_sections) -> Tensor(a)[]
```

Comment:
Could we rename all of these `indices_or_sections` variables to either `indices` (when it's `int[]`) or `sections` (when it's int)? Also, the functions in `aten/src/ATen/native/TensorShape.cpp` should match these argument names.

Reply:
Note, I think `indices_or_sections` is still a good name for the Python interface.